### PR TITLE
Update help text for add-unit command.

### PR DESCRIPTION
**Pull request by mitechie**
- Update to match the docs standard
- Clean up some repeat text
- make examples consistent with mysql vs mix mysql/mariadb

### CODE REVIEW

File: cmd/juju/application/addunit.go [hunk 1]
```diff
@@ -24,51 +24,54 @@ var usageAddUnitSummary = `
...
+
+Add a unit of mysql to LXD container on a new machine:
 
-Add a unit of mariadb to LXD container on a new machine:
-    juju add-unit mariadb --to lxd
+    juju add-unit mysql --to lxd
 
-See also: 
+See also:
     remove-unit`[1:]
```
Comment (hmlanigan) [inline cmd/juju/application/addunit.go]: wallyworld always gripes at me that `[1:] belongs on a new line for the long string vars.  :-)

File: cmd/juju/application/addunit.go [hunk 2]
```diff
@@ -24,51 +24,54 @@ var usageAddUnitSummary = `
 Adds one or more units to a deployed application.`[1:]
```
Comment (hmlanigan) [inline cmd/juju/application/addunit.go]: `[1:] on new line pls.

File: cmd/juju/application/addunit.go [hunk 3]
```diff
@@ -24,51 +24,54 @@ var usageAddUnitSummary = `
...
     juju add-unit mysql --to 23
 
 Add two units of mysql to machines 3 and 4:
-   juju add-unit mysql -n 2 --to 3,4
 
-Add three units of mysql to machine 7:
-    juju add-unit mysql -n 3 --to 7,7,7
+   juju add-unit mysql -n 2 --to 3,4
 
 Add three units of mysql, one to machine 3 and the others to new
```
Comment (hmlanigan) [inline cmd/juju/application/addunit.go]: the --to directive indicates machine 7 not 3.

File: cmd/juju/application/addunit.go [hunk 4]
```diff
@@ -24,51 +24,54 @@ var usageAddUnitSummary = `
...
 
 Add five units of wordpress on five new machines:
+
     juju add-unit wordpress -n 5
 
 Add a unit of mysql to machine 23 (which already exists):
+
     juju add-unit mysql --to 23
 
 Add two units of mysql to machines 3 and 4:
```
Comment (hmlanigan) [inline cmd/juju/application/addunit.go]: do machines 3 & 4 already exist?  is mentioned in example above.

Description seems ambiguous?  "Add two units of mysql, one each, to machines 3 and 4:"?

File: cmd/juju/application/addunit.go [hunk 5]
```diff
@@ -24,51 +24,54 @@ var usageAddUnitSummary = `
...
-Add three units of mysql to machine 7:
-    juju add-unit mysql -n 3 --to 7,7,7
+   juju add-unit mysql -n 2 --to 3,4
 
 Add three units of mysql, one to machine 3 and the others to new
 machines:
+
     juju add-unit mysql -n 3 --to 7
 
 Add a unit into a new LXD container on machine 7:
```
Comment (hmlanigan) [inline cmd/juju/application/addunit.go]: "add a unit of mysql to a new...."?

File: cmd/juju/application/addunit.go [hunk 6]
```diff
@@ -24,51 +24,54 @@ var usageAddUnitSummary = `
...
+
     juju add-unit mysql --to lxd:7
 
 Add two units into two new LXD containers on machine 7:
+
```
Comment (hmlanigan) [inline cmd/juju/application/addunit.go]: "add two units of mysql to two new..."?

File: cmd/juju/application/addunit.go [hunk 7]
```diff
@@ -24,51 +24,54 @@ var usageAddUnitSummary = `
...
-machines or containers, which will bypass application and model
-constraints.
+By default, units are deployed to newly provisioned machines in accordance
+with any application or model constraints. This command also supports the
+placement directive ("--to") for targeting specific machines or containers,
+which will bypass application and model constraints.
 
 Examples:
 
 Add five units of wordpress on five new machines:
```
Comment (hmlanigan) [inline cmd/juju/application/addunit.go]: Did you want to be consistent with mysql?  This one still uses wordpress.

File: cmd/juju/application/addunit.go (hunk 1 above)
Comment (mitechie) [reply cmd/juju/application/addunit.go]: Makes sense, thanks for the heads up.

File: cmd/juju/application/addunit.go (hunk 7 above)
Comment (mitechie) [reply cmd/juju/application/addunit.go]: +1, updated

File: cmd/juju/application/addunit.go (hunk 6 above)
Comment (mitechie) [reply cmd/juju/application/addunit.go]: ty

File: cmd/juju/application/addunit.go (hunk 5 above)
Comment (mitechie) [reply cmd/juju/application/addunit.go]: ty

File: cmd/juju/application/addunit.go (hunk 4 above)
Comment (mitechie) [reply cmd/juju/application/addunit.go]: Thanks, noted that they're "existing machines"

File: cmd/juju/application/addunit.go (hunk 3 above)
Comment (mitechie) [reply cmd/juju/application/addunit.go]: ty

File: cmd/juju/application/addunit.go (hunk 2 above)
Comment (mitechie) [reply cmd/juju/application/addunit.go]: rgr

**Comment by mitechie** [comment]
Thanks for the review! $$merge$$

**Pull request merged**